recovery.go

```diff
@@ -1,42 +0,0 @@
-package main
-
-import (
-	"net/http"
-	"runtime"
-
-	"github.com/raintank/worldping-api/pkg/log"
-)
-
-type recoveryHandler struct {
-	handler http.Handler
-}
-
-// RecoveryHandler is HTTP middleware that recovers from a panic,
-// logs the panic as well as details about the request
-// and writes http.StatusInternalServerError.
-func RecoveryHandler(h http.Handler) http.Handler {
```

Comment:
This handled my macaron. (we use macaron.Classic())
https://go-macaron.com/docs/intro/core_concepts#classic-macaron